VTEX - Pick and Pack Last Mile Protocol API.json

```diff
@@ -1402,88 +1904,320 @@
                             }
                         }
                     },
-                    "200": {
-                        "description": "OK",
-                        "content": {
-                            "application/json; charset=utf-8": {
-                                "schema": {
-                                    "$ref": "#/components/schemas/ServiceBody6"
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json; charset=utf-8": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/ServiceBody6"
+                                },
+                                "example": {
+                                    "message": "Service on hold successfully",
+                                    "data": {
+                                        "comment": "The delivery address was changed",
+                                        "type": "END_CLIENT_PAUSE"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                "deprecated": false,
+                "servers": [
+                    {
+                        "url": "https://api.pickingnpacking.com/{environment}/v1/tracking/hook",
+                        "variables": {
+                            "environment": {
+                                "default": "DefaultParameterValue"
+                            }
+                        }
+                    }
+                ]
+            }
+        }
+    },
+    "components": {
+        "parameters": {
+            "Content-Type": {
+                "name": "Content-Type",
+                "in": "header",
+                "description": "Type of the content being sent.",
+                "required": true,
+                "style": "simple",
+                "schema": {
+                    "type": "string",
+                    "example": "application/json"
+                }
+            },
+            "Accept": {
+                "name": "Accept",
+                "in": "header",
+                "description": "HTTP Client Negotiation _Accept_ Header. Indicates the types of responses the client can understand.",
+                "required": true,
+                "style": "simple",
+                "schema": {
+                    "type": "string",
+                    "example": "application/json"
+                }
+            },
+            "Authorization": {
+                "name": "Authorization",
+                "in": "header",
+                "description": "Authentication JWT.",

```

Comment:
Daria mais alguma informação sobre JWT. Nesta [documentação](https://developers.vtex.com/docs/guides/vtex-pick-and-pack-get-order-status-and-information-api) de Pick and Pack está: 

JSON Web Token (JWT) is an open standard ([RFC 7519](https://www.rfc-editor.org/rfc/rfc7519)) that defines a compact and self-contained way for securely transmitting information as a JSON object between parties.